packages/core/src/lib/user-organization/user-organization.services.ts

```
@@ -30,8 +30,12 @@
 		filter: BaseQueryDTO<UserOrganization>,
 		includeEmployee: boolean
 	): Promise<IPagination<UserOrganization>> {
-		// Call the base class method to find all user organizations
-		let { items, total } = await super.findAll(filter);
+		// Prepare filter with appropriate permissions
+		// Ensure we never pass an undefined filter to the helper
+		const modifiedFilter = this.applySensitiveRelationsFilter(filter ?? ({} as BaseQueryDTO<UserOrganization>));
+
+		// Execute the query with the potentially modified relations
+		let { items, total } = await super.findAll(modifiedFilter);
```

Biome (1.9.4)

[error] This let declares some variables that are only assigned once.

'items' is never reassigned.

'total' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)